src/flp/types/fixedpoint_l2.rs

```diff
@@ -606,8 +606,122 @@ mod tests {
     use fixed::{FixedI128, FixedI16, FixedI64};
     use fixed_macro::fixed;
 
+    fn test_fixed<F: Fixed>(fp_vec: Vec<F>, enc_vec: Vec<u128>)
```

Comment:
Can we put this code back where it was so that we can review the changes line-by-line?